api/iceberg-service/src/main/java/org/apache/polaris/service/types/NotificationType.java

```diff
@@ -84,9 +83,9 @@ public static Optional<NotificationType> lookupByName(String name) {
       return Optional.empty();
     }
 
-    for (NotificationType NotificationType : NotificationType.values()) {
-      if (name.toUpperCase(Locale.ROOT).equals(NotificationType.name())) {
-        return Optional.of(NotificationType);
+    for (NotificationType notificationType : NotificationType.values()) {
+      if (name.equalsIgnoreCase(notificationType.name())) {
```

Comment:
Is this flagged by ErrorProne or collateral?

Reply:
collateral as i thought it didnt make sense to always call uppercase inside the loop.
since the method is unused we can still remove all of it.